### PR TITLE
Feature/pst orbital elements

### LINKED ISSUE #9: Create a Python script to visualize satellite orbits
<h1> Description </h1>
The objective here is to write a python toolset to illustrate a general satellite orbit. The python script should enable the user to compare the positions and orbits of GNSS-RO sender and receiver.

<h2> Requirements </h2>
The script needs to be able to do the following:

- [ ] Plot the earth at the coordinate center *to scale*.
- [ ] Plot one or more satellite orbits in 3D *to scale*.

<h2> Optional </h2>

Optional requirements are the following:

- [ ] Create the above scene with an animated satellite position as a moving dot.

### LINKED ISSUE #10: Write a TLE parser
<h1> Description </h1>

Write a python script that downloads and parses data in TLE format.

### PR DESCRIPTION
**Pull request by StegmannJCSDA**
<h1> Description </h1>
This PR adds Python helper scripts to process Two-Line-Element data of satellite orbits.
The scripts do the following:

- Download the TLE text files from `http://www.celestrak.com/NORAD/elements/`.
- Parse the TLE text into human-readable information.
- Plot satellite orbits based on TLE data.

<h1> Connected Issues: </h1>

- closes #10 
- closes #9 

<h1> Dependencies: </h1>
N/A

### CODE REVIEW

File: scripts/python/main.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    for urlname in names:
+        f = urllib.request.urlopen("http://www.celestrak.com/NORAD/elements/%s.txt" % urlname)
+        elem = ""
+        for line in f:
+            line = str(line.decode('UTF-8'))
+            print(line)
+            elem += line
+            if (line[0] == '2'):
+                elem = elem.strip()
+                print(elem)
```
Comment (lliu1220) [inline scripts/python/main.py]: was printing the following:

GPS BIII-3  (PRN 23)
1 45854U 20041A   22038.44237535 -.00000055  00000+0  00000+0 0  9990
2 45854  55.4238  91.4915 0019468 175.7803 312.9411  2.00560548 12131
GPS BIII-4  (PRN 14)

1 46826U 20078A   22038.03999833 -.00000014  00000+0  00000+0 0  9996

2 46826  54.6537 275.3858 0013962 182.5691 347.5095  2.00566642  9596

GPS BIII-4  (PRN 14)
1 46826U 20078A   22038.03999833 -.00000014  00000+0  00000+0 0  9996
2 46826  54.6537 275.3858 0013962 182.5691 347.5095  2.00566642  9596

File: scripts/python/main.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                if elem.startswith("GPS BIIR-2"):
+                    gps_orbit.inspect(elem, printInfo=True, labels=True)
+                elem = ""
+
+    # gps_orbit.inspect(elem1)
+    # gps_orbit.inspect(dragon)
+
+    orbit_graphics.doDraw()
```
Comment (lliu1220) [inline scripts/python/main.py:67]: I had Xming open, but this line didn't make graphics for my case.
Can you use matplotlib and save the figure maybe?

File: scripts/python/main.py (hunk 1 above)
Comment (StegmannJCSDA) [reply scripts/python/main.py]: Right. I have cleaned up the print statements to reduce the output clutter.

File: scripts/python/main.py (hunk 2 above)
Comment (StegmannJCSDA) [reply scripts/python/main.py:67]: I have added a line to save the plot as `orbit_plot.png`.

Comment (lliu1220) [reply scripts/python/main.py:67]: woo-hoo!
image

**Review by lliu1220 — APPROVED** [review]

File: scripts/python/main.py (hunk 1 above)
Comment (lliu1220) [reply scripts/python/main.py]: Nice and clean!

semi_major_axis = 26560.4km
eccentricity    = 0.0056176
inclination     = 55.4949°
arg_perigee     = 53.8316°
right_ascension = 160.773°
true_anomaly    = -279.751°
----------------------------------------------------------------------------------------
GPS BIIR-2  (PRN 13) : Lat: -48.3961° Long: -50.7274
GPS BIIR-2  (PRN 13)
1 24876U 97035A   22037.73434929  .00000020  00000+0  00000+0 0  9996
2 24876  55.4949 160.7731 0056176  53.8316 306.6658  2.00563345180035

**Pull request merged**